handout/routing/routeroutlet.md

```diff
@@ -22,6 +22,6 @@ export class AppComponent {}
 
 In the above example, the component corresponding to the route specified will be placed after the `<router-outlet></router-outlet>` element when the link is clicked.
 
-[View Example](https://plnkr.co/edit/odsZq1pL43HtWcUr0taJ?p=preview)
+[View Example](https://plnkr.co/edit/OHfytJquXKm8jvSe2T9Y?p=preview)
```

Comment:
We used to have a link on here that would open the example in a new window and actually allow the routes to change.  Can we get that working again?  I was a little surprised by the `[routerLink]` -> `routerLink` change and I wanted to see it in action.